### PR TITLE
6 implement redirects

### LINKED ISSUE #6: Implement redirects
Website redirects

- [ ] Root domain should redirect to /en

### PR DESCRIPTION
**Pull request by mkbansal1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #6 

Test URLs:
- Before: https://main--fonterra-fernleaf--hlxsites.hlx.page/
- After: https://6-implement-redirects--fonterra-fernleaf--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,14 +1,58 @@
...
 }
 
-footer .footer p {
+footer ul {
+  list-style: none;
+  padding: 0;
   margin: 0;
+}
+
+@media only screen and (max-width: 1023px) {
```
Comment (asthabh23) [inline blocks/footer/footer.css:17]: As discussed in our last call, the media breakpoints should considerably be 600px, 900px, 1200px

File: styles/favicon.ico
Comment (asthabh23) [inline styles/favicon.ico:1]: this should go into the root of the repo

File: styles/favicon.png
Comment (asthabh23) [inline styles/favicon.png:1]: only ico file for favicon is required, this can be removed

**Review by asthabh23 — CHANGES_REQUESTED** [review]
The before and after URLs need to be updated to /en ones

File: blocks/footer/footer.css (hunk 1 above)
Comment (mkbansal1) [reply blocks/footer/footer.css:17]: @asthabh23  is this agreed with client?

File: styles/favicon.ico
Comment (mkbansal1) [reply styles/favicon.ico:1]: @asthabh23  other repos also have fav icon in styles folder.

File: blocks/footer/footer.css (hunk 1 above)
Comment (mkbansal1) [reply blocks/footer/footer.css:17]: Done

File: styles/favicon.png
Comment (mkbansal1) [reply styles/favicon.png:1]: Removed .png

File: styles/favicon.ico
Comment (asthabh23) [reply styles/favicon.ico:1]: not sure on this, but based on a recent change, having favicon.ico file in root of GH repo directly updates the favicon for the site. For the current repo itself (https://github.com/hlxsites/fonterra-fernleaf), you can see default favicon.ico file at the root. You would just need to replace it with the new one.

**Comment by mkbansal1** [comment]
@asthabh23  i have incorporated the feedback, please approve this.

File: styles/favicon.ico
Comment (mkbansal1) [reply styles/favicon.ico:1]: Ok, moved it to root.

**Review by asthabh23 — APPROVED** [review]

**Pull request merged**